### ISSUE TITLE
Which dataset you used for training and what is performance of this implementation

### ISSUE THREAD

**Issue by humayun** [Fei-Wang/insightface#7]
Dear,

insightface used multiple dataset like faces_emore or MS1M_Celab etc to training and validation accuracy. Can you explicitly mentioned that which dataset you tried and what are final loss and accuracy on training and different validation sets. 
If you can update the links and information of dataset on your readme page, that would be extremely helpful to start working on this code if someone who never work on face recognition problem.

Best
